wp-config.php

```diff
@@ -77,7 +77,7 @@
  *
  * @link https://codex.wordpress.org/Debugging_in_WordPress
  */
-define('WP_DEBUG', false);
```

Comment:
@ferrerluis this is why when you got it working, nothing showed up. should be changed in production though